portworx/README.md

```diff
@@ -121,3 +121,27 @@ Global Storage Pool
     Total Capacity    :  300 GiB
 ```
 
+## Deleting a Portworx cluster
+
+Since deleting a Portworx cluster implies the deletion of data, the cluster-delete operation is multi-step, to ensure operator intent.
+
+### Ensure the desired context
+
+Make sure your **KUBECONFIG** environment variable points to the Kubernetes cluster you intend to target.
+
+### Delete Helm chart
+
+Run 'helm list' to find the name of the Helm chart correpsonding to Portworx.
+Then perform a 'helm delete' of the corresponding chart.
+At this point, the Portworx `Daemonset` has been removed from Kubernetes.   But the nodes cannot (yet) be reinstalled with another Portworx cluster.
+No user data has been destroyed at this point.
+
+### Wipe Portworx cluster
+
+In order to cleanly re-install Portworx after a previous installation, the cluster will have to be **"wiped"**
+Issue the following command:
+
+```
+     curl https://install.portworx.com/px-wipe | bash
```

Comment:
As per my discussion with Sen yesterday, he said we should execute this curl command to wipe out first without deleting the Portworx daemon set. Else the wipe out will not happen. Also this curl will do wipe-out and also delete all kubernetes resources of Portworx so technically we don't need helm delete. Or users can do helm delete after curl which means basically nothing is there is delete at that point.